vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -71,14 +66,34 @@ mkdir bundle
 ```
 2. Run the bundler:
 ``` sh
-ent bundler from-env --location bundle --code YOUR-TEST-BUNDLE --description “Your Test Bundle”
+ent bundler from-env --location bundle --code YOUR-BUNDLE-NAME --description “Your Exported Bundle”
 ```
-The bundler will inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files and assemble the top-level descriptor file.
+The bundler will inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files, and assemble the top-level descriptor file.
 
 ``` 
 $ ls bundle
 assets      contentModels  contents         fragments  labels     pageModels  resources
 categories  contentTypes   descriptor.yaml  groups     languages  pages       widgets
 ```
 
-You now have a complete Entando project structure! You can inspect the output to edit the exported components or deploy it to another Entando Application.
+You now have a complete Entando project structure for a git-based bundle! You can inspect the output to edit the exported components or [deploy](publish-project-bundle.md) it to another Entando Application. 
```

Comment:
maybe i'm misunderstanding the use case. @nshaw  if you have a bundle with some good stuff in it and want a similar, slightly different bundle in the same application, could/should you use the export command instead of locally cloning the files and doing surgery on what you want to keep/toss? if this is a possibility i suggest we remove the word "another"